package/MDAnalysis/coordinates/base.py

```diff
@@ -2135,6 +2150,13 @@ def __del__(self):
             self._auxs[aux].close()
         self.close()
 
+    def __getstate__(self):
+        return self.__dict__
```

Comment:
Isn't this just what `__getstate__` is by default?

Reply:
I was not sure `__setstate__` works without `__getstate__`. Seems that it does. I will remove this part.